articles/app-service/includes/quickstart-php/quickstart-php-azure-portal-pivot.md

```diff
@@ -0,0 +1,200 @@
+[Azure App Service](../../overview.md) provides a highly scalable, self-patching web hosting service.  This quickstart tutorial shows how to deploy a PHP app to Azure App Service on Linux using the Azure portal.
+
+This quickstart configures an App Service app in the **Free** tier and incurs no cost for your Azure subscription.
+
+This quickstart shows you how to make these changes within your browser, without having to install the development environment tools on your machine.
+
+![Sample app running in Azure](../../media/quickstart-php/php8-hello-world-in-browser.png)
+
+You can follow the steps here using a Mac, Windows, or Linux machine. Once the prerequisites are installed, it takes about five minutes to complete the steps.
+
+[!INCLUDE [quickstarts-free-trial-note](../../../../includes/quickstarts-free-trial-note.md)]
+
+## Prerequisites
+
+- Have an Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?utm_source=campaign&utm_campaign=vscode-tutorial-app-service-extension&mktingSource=vscode-tutorial-app-service-extension).
+- Have a GitHub account to fork a repository.
+
+## Fork the sample repository
```

Comment:
we're using a new formatting to make H2s more readable. Please use "1 - Fork the sample repository", and so on.

Reply:
@cephalin I have updated the H2s after the Prerequisites H2. Should that be numbered as well?

Reply:
@cephalin Disregard the prereqs question. I reworked the prereqs to follow [the Python quickstart](https://docs.microsoft.com/azure/app-service/quickstart-python), as I know that has the new H2s in it.